openai/openai_test.go

```diff
@@ -318,10 +319,13 @@ func TestChatMiddleware(t *testing.T) {
 				}
 			}
 			if capturedRequest != nil && !reflect.DeepEqual(tc.req, *capturedRequest) {
+				t.Logf("  diff: %+v", cmp.Diff(tc.req, *capturedRequest))
 				t.Fatal("requests did not match")
 			}
```

Comment:
nit: this can be simplified
```suggestion
			if diff := cmp.Diff(tc.req, *capturedRequest); diff != "" {
				t.Fatalf("requests did not match: %+v", diff)
			}
```